src/components/PeopleTable/SegmentAssignmentModal.jsx

```diff
@@ -0,0 +1,246 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  chakra,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Flex,
+  HStack,
+  VStack,
+  Box,
+  Button,
+  Text,
+  Tag,
+  TagLabel,
+  TagCloseButton,
+} from '@chakra-ui/react';
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import { chakraComponents, Select } from 'chakra-react-select';
+
+const util = require('util');
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'util' is assigned a value but never used.